src/components/Inputs/CustomComboBox.js

```diff
@@ -138,6 +139,7 @@ const CustomComboBox = ({
         <TextField
           {...params}
           type={type}
+          onKeyDown={(e) => !editable && e.preventDefault()}
```

Comment:
Would this prevent us from clicking tab?

Reply:
yes it affects this but after additional change it is fixed now